doc/man3/SSL_get0_peer_rpk.pod

```diff
@@ -0,0 +1,108 @@
+=pod
+
+=head1 NAME
+
+SSL_add_expected_rpk,
+SSL_get_negotiated_client_cert_type,
+SSL_get_negotiated_server_cert_type,
+SSL_get0_peer_rpk,
+SSL_SESSION_get0_peer_rpk - raw public key (RFC7250) support
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ int SSL_add_expected_rpk(SSL *s, EVP_PKEY *rpk, int nid);
+ int SSL_get_negotiated_client_cert_type(const SSL *s);
+ int SSL_get_negotiated_server_cert_type(const SSL *s);
+ EVP_PKEY *SSL_get0_peer_rpk(const SSL *s);
+ EVP_PKEY *SSL_SESSION_get0_peer_rpk(const SSL_SESSION *ss);
+
+=head1 DESCRIPTION
+
+SSL_add_expected_rpk() adds public key B<rpk> into the SSL B<s>'s DANE TLSA database.
+The B<nid> argument specifies the hash method to use.
+
+SSL_get_negotiated_client_cert_type() returns the negotiated client certificate
+type used on a conection represented by B<s>.
+
+SSL_get_negotiated_server_cert_type() returns the negotiated server certificate
+type used on a conection represented by B<s>.
+
+SSL_get0_peer_rpk() returns the peer's raw public key from SSL B<s>.
+
+SSL_SESSION_get0_peer_rpk() returns the peer's raw public key from
+SSL_SESSION B<ss> in the case of resumption.
+
+=head1 NOTES
+
+Raw public keys are used in place of certificates when the option is
+negotiated. Adding more than one key is possible to allow for key rotation,
```

Comment:
```suggestion
negotiated.
Adding more than one expected public key is possible to allow for key rotation,
```
(Note, mandoc strongly recommends that each sentence begin a new line, so to get compliant manpages always add a linebreak at each sentence boundary).

Reply:
Not all the OSSL documentation follows that convention. I'd rather focus on the code...